autolens/plot/plotters/fit_imaging_plotters.py

```diff
@@ -0,0 +1,292 @@
+        - border: the border of the structure's mask.
+
+        Parameters
+        ----------
+        structure : abstract_structure.AbstractStructure
+            The structure whose attributes are extracted for plotting.
+
+        Returns
+        -------
+        vis.Visuals2D
+            The collection of attributes that can be plotted by a `Plotter2D` object.
+        """
+
+        visuals_2d = super(FitImagingPlotter, self).visuals_with_include_2d
+
+        visuals_2d.mask = None
+
+        return visuals_2d + lensing_visuals.Visuals2D(
+            light_profile_centres=self.extract_2d(
+                "light_profile_centres", self.tracer.planes[0].light_profile_centres
+            ),
+            mass_profile_centres=self.extract_2d(
+                "mass_profile_centres", self.tracer.planes[0].mass_profile_centres
+            ),
+            critical_curves=self.extract_2d(
+                "critical_curves", self.tracer.critical_curves
+            ),
+        )
+
+    @property
+    def tracer(self):
+        return self.fit.tracer
+
+    @property
+    def tracer_plotter(self):
+        return ray_tracing_plotters.TracerPlotter(
+            tracer=self.tracer,
+            grid=self.fit.grid,
+            mat_plot_2d=self.mat_plot_2d,
+            visuals_2d=self.visuals_2d,
+            include_2d=self.include_2d,
+        )
+
+    @property
+    def inversion_plotter(self):
+        return inversion_plotters.InversionPlotter(
+            inversion=self.fit.inversion,
+            mat_plot_2d=self.mat_plot_2d,
+            visuals_2d=self.tracer_plotter.visuals_with_include_2d_of_plane(
+                plane_index=1
+            ),
+            include_2d=self.include_2d,
+        )
+
+    @abstract_plotters.for_figure_with_index
+    def figure_subtracted_image_of_plane(self, plane_index):
+        """Plot the model image of a specific plane of a lens fit.
+    
+        Set *autolens.datas.arrays.plotter.plotter* for a description of all input parameters not described below.
+    
+        Parameters
+        -----------
+        fit : datas.fitting.fitting.AbstractFitter
+            The fit to the datas, which includes a list of every model image, residual_map, chi-squareds, etc.
+        image_index : int
+            The index of the datas in the datas-set of which the model image is plotted.
+        plane_indexes : int
+            The plane from which the model image is generated.
+        """
+
+        if self.tracer.total_planes > 1:
+
+            other_planes_model_images = [
+                model_image
+                for i, model_image in enumerate(self.fit.model_images_of_planes)
+                if i != plane_index
+            ]
+
+            subtracted_image = self.fit.image - sum(other_planes_model_images)
+
+        else:
+
+            subtracted_image = self.fit.image
+
+        self.mat_plot_2d.cmap.kwargs["vmin"] = np.max(
+            self.fit.model_images_of_planes[plane_index]

```

Comment:
Take the min or max on one line and assign the variable on another. It is _slightly_ more concise